packages/e2e-tests/plugins/block-bindings/index.js

```diff
@@ -46,3 +51,43 @@ registerBlockBindingsSource( {
 	getValues,
 	canUserEditValue: () => true,
 } );
+
+const withBlockBindingsInspectorControl = createHigherOrderComponent(
+	( BlockEdit ) => {
+		return ( props ) => {
+			if ( ! props.attributes?.metadata?.bindings?.content ) {
+				return el( BlockEdit, props );
+			}
```

Comment:
To avoid affecting other tests where `content` is connected, maybe it is safer to provide a unique name for this specific test? For example, checking `props.attributes?.metadata?.bindings === 'bindings-inspector-controls'`.

Reply:
Can you explain how this generic UI control could affect other e2e tests and what the idea is to prevent it? `bindings` is an object so I'm not sure how to compare that with a string.

Reply:
Sorry, I meant `metadata?.name`, not `metadata?.bindings`.

The idea behind it was that right now it is adding the extra control for all the tests with content bound. I guess it shouldn't affect, but I thought it could be safer to provide an unique property from tests that need the extra control and check against that instead.

Reply:
What are other conditions that would get closer to real-life usage? I don't think extenders would ever target their integrations based on the name provided by the user through UI for the block name.